src/pipeline/activator/activator/treehealth/main.py

```diff
@@ -0,0 +1,251 @@
+
+
+def collection_id_to_collection_id(aviris_collection_id):
+    if aviris_collection_id == 'aviris-l1-cogs':
+        collection_id = 'aviris-treehealth-l1-cogs'
+        description = 'Tree Health Predictions (L1)'
+    elif aviris_collection_id == 'aviris-l2-cogs':
+        collection_id = 'aviris-treehealth-l2-cogs'
+        description = 'Tree Health Preditions (L2)'
+    else:
+        raise Exception(f'Unrecognized collection id: {aviris_collection_id}')
+    return collection_id, description
+
+
+def get_aviris_cog_collection(aviris_collection_id):
+    collection_id, description = collection_id_to_collection_id(
+        aviris_collection_id)
+    collection = pystac.Collection(collection_id,
+                                   description,
+                                   pystac.Extent(
+                                       pystac.SpatialExtent(
+                                           [[-180, -90, 180, 90]]),
+                                       pystac.TemporalExtent([[
+                                           datetime(2014, 1, 1, tzinfo=timezone.utc),
+                                           datetime(2020, 1, 1, tzinfo=timezone.utc),
+                                       ]]),
+                                   ),
+                                   stac_extensions=COG_COLLECTION_EXTENSIONS)
+    collection.links = []
+    collection.properties = {}
+    collection.properties['eo:bands'] = [
+        {
+            'B1': 'Red-Stage Conifer'
+        },
+        {
+            'B2': 'Green-Stage Conifer'
+        },
+        {
+            'B3': 'Non-Conifer'
+        },
+    ]
+
+    return collection
+
+
+def activation_output(item_id: str, collection_id: str):
+    with open('/tmp/activator-output.json', 'w') as outfile:
+        json.dump(
+            {
+                'sourceCollectionId': collection_id,
+                'sourceItemId': item_id
+            }, outfile)
+
+
+def pipeline_arguments(args):
+    def to_snake(word):
+        return ''.join([
+            '_' + letter.lower() if letter.isupper() else letter
+            for letter in word
+        ])
+
+    filename_pipeline = None
+    if args.pipeline_uri:
+        filename_pipeline = '/tmp/pipeline.json'
+        os.system(f'aws s3 cp {args.pipeline_uri} {filename_pipeline}')
+    elif args.pipeline:
+        filename_pipeline = args.pipeline
+    if filename_pipeline is not None:
+        with open(filename_pipeline, 'r') as file:
+            args_json = json.loads(file.read().replace('\n', ''))
+        for k in args_json:
+            k2 = to_snake(k)
+            setattr(args, k2, args_json.get(k))
+    return args
+
+
+def cli_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--aviris-collection-id", type=str, default=AVIRIS_ARCHIVE_COLLECTION_ID)
+    parser.add_argument("--aviris-stac-id", type=str, default="f140603t01p00r18_sc01", help="STAC Item ID to process from the STAC collection")
```

Comment:
Should not be this abstracted over the sensor type, or you're pretty sure that it is very AVRIS specific?

Reply:
This is decidedly AVIRIS specific.  Can revisit later if we add models for other imagery types.